ingenialink/pdo.py

```diff
@@ -202,28 +211,61 @@ class PDOMap:
     def __init__(self) -> None:
         self.__items: list[PDOMapItem] = []
         self.__map_register_address: Optional[int] = None
+        self.__slave = None
+
+    @property
+    def slave(self) -> Optional["CdefSlave"]:
```

Comment:
I think that the idea of this separate module was to make it generic, for ethercat and canopen idependently.
Canopen is not yet implement it, but can we try to find a way to keep it generic?

Some ideas I have not tried:
What if slave is PdoServo instead?,
can we put check_servo_is_not_in_operational_state on PdoServo as an abstract method, and then implement it on EthercatNetwork for pysoem?